tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```
@@ -355,6 +357,8 @@ public ImmutableTracerSettings(TracerSettings settings)
         /// </summary>
         internal string TraceMethods { get; }
 
+        internal string ProfilerExportType { get; }
```

Consider to make it enum. You can then parse it while creating settings so you can fail fast/logs warning than the default value should be used.